python/paddle/fluid/framework.py

```diff
@@ -992,7 +993,7 @@ def convert_np_dtype_to_dtype_(np_dtype):
         return core.VarDesc.VarType.INT64
     elif dtype == np.bool:
         return core.VarDesc.VarType.BOOL
-    elif dtype == np.uint16:
+    elif dtype == bfloat16:
         # since there is still no support for bfloat16 in NumPy,
         # uint16 is used for casting bfloat16
         return core.VarDesc.VarType.BF16
```

Comment:
Hi, Do you think if it is necessary to map np.uint16 to core.VarDesc.VarType.UINT16?  Or make it reserved datatype? Also please modify the comments ?
@Aganlengzi  Please review

Reply:
Hi, this PR is not ready yet, I have made it just to see which tests and CIs will fail

Reply:
Hi, ok, thanks